docs/adding-new-storage-adapter.md

```diff
@@ -0,0 +1,57 @@
+# Adding New Storage Adapter
+
+This document is a part of Utopia PHP contributors' guide. Before you continue reading this document make sure you have read the [Code of Conduct](../CODE_OF_CONDUCT.md) and the [Contributing Guide](../CONTRIBUTING.md).
+
+## Getting Started
+
+Storage providers help us use various storage services to store our data. As of the writing of these lines we already support Local storage, [AWS S3](https://aws.amazon.com/s3/) storage and [Digitalocean Spaces](https://www.digitalocean.com/products/spaces/) storage.
+
+## 1. Prerequisities
+
+It's really easy to contribute to an open source project, but when using GitHub, there are a few steps we need to follow. This section will take you step-by-step through the process of preparing your own local version of Utopia PHP Storage, where you can make any changes without affecting Utopia PHP Storage right away.
+
+> If you are experienced with GitHub or have made a pull request before, you can skip to [Implement new Storage Adapter](#2-implement-new-storage-adapter).
+
+###  1.1 Fork the Appwrite repository
+
+Before making any changes, you will need to fork Utopia PHP Storage's repository to keep branches on the official repo clean. To do that, visit the [Utopia PHP Storage Github repository](https://github.com/utopia-php/storage) and click on the fork button.
+
+This will redirect you from `github.com/utopia-php/storage` to `github.com/YOUR_USERNAME/storage`, meaning all changes you do are only done inside your repository. Once you are there, click the highlighted `Code` button, copy the URL and clone the repository to your computer using `git clone` command:
```

Comment:
```suggestion
This will redirect you from `github.com/utopia-php/storage` to `github.com/YOUR_USERNAME/storage`, meaning all further changes will reflect on your copy of the repository. Once you are there, click the highlighted `Code` button, copy the URL and clone the repository to your computer using `git clone` command:
```